### DISCUSSION TITLE
Schaltzeiten Zeitplan

### DISCUSSION THREAD

**Discussion by HerwigHahn** [camueller/SmartApplianceEnabler#559] [Installation und Betrieb]
Gerne würde ich die Wallbox in der Nacht kurz nach Mitternacht aktivieren und dann (in der Zeit bis 6:00 Uhr morgens max. 20 kW laden.
Ich habe u.s. Zeitpan erstellt, der aber nicht funktioniert. Was mir aufgefallen ist, ist dass ich bei mir nicht die Möglichkeit habe, eine Min/Max Laufzeit anzugeben (felder fehlen). Liegt das daran?
![grafik](https://github.com/camueller/SmartApplianceEnabler/assets/146930279/088459e1-1d49-4627-a224-41c4460d6ac1)

**Comment by camueller** [comment]
Du hast ja (richtigerweise) als "Anforderungsart" den Wert "Energie" ausgewählt. Als werden hier nur min/max-Wert für die Energie vorgegeben und keine Laufzeit. Dein Zeitplan sieht eigentlich korrekt aus.

Ich selbst nutze das Zeitplanbasierte Laden nicht, aber ich kann mal testen, ob das bei mir funktioniert.

**Comment by HerwigHahn** [comment]
Danke für die Rückmeldung. 

Die WB scheint angesteuert zu werden, aber folgende Meldung liegt am Morgen vor.

**Comment by camueller** [comment]
Wer zeigt denn welche Meldung wo an???

**Comment by camueller** [comment]
Bei meinen Tests mit Zeitplanbasiertem Laden konnte ich einen Fehler im SAE finden. Muss ihn allerdings noch beheben ...

**Comment by HerwigHahn** [comment]
Danke fürs Prüfen und gelegentliches Beheben. 
Mit Meldung meinte ich folgende Stautusanzeige: 
 
![grafik](https://github.com/camueller/SmartApplianceEnabler/assets/146930279/a3df4eed-ef1b-4134-b2ad-7c8ee6cb5790)
